backend/de.metas.banking/de.metas.banking.base/src/main/sql/postgresql/system/5681940_gh14909_avoid_having_payment_with_0_openamt.sql

```diff
@@ -84,3 +84,10 @@ COMMENT ON FUNCTION getopenpayments(
 * Used in de.mets.paymentallocation.form.Allocation.queryPaymentTable()
 * Uses the view T_GetOpenPayments as return type'
 ;
+
+
+ALTER FUNCTION paymentallocatedamt(numeric) STABLE
+;
+
+ALTER FUNCTION paymentavailable(numeric) STABLE
```

Comment:
pls also update 
backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/paymentAllocatedAmt.sql
backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/paymentAvailable.sql

if not, the next dev will have to change those functions, will loose the `STABLE` feature